README.md

```diff
@@ -0,0 +1,41 @@
+# GovernmentBillSummarizer
+
+Use NLP to classify and summarize bills available through the [govinfo API](https://api.govinfo.gov/docs/).
+
+## Using
+
+To run program correctly, the program must be initialized by running `UserInterface.py`:
+
+```bash
+python UserInterface.py
+```
+
+### Additional Notes
+
+- For *package bills*, the government api sorts these by *upload date* and not issue date.
+
+  If you are searching for a bill from a certain date, this will not find it, you must use general bills.
+
+  General bills can be searched by bill issue/creation date.
+
+- Some very large bills summaries take a few minutes to load, if it seems like the program hangs up, it is probably loading a summary so be patient.
+
+- Some summaries have repeated sentences, this is a known issue, and is due to quirks in the .gov api.
+
+- All folder's (train and test folder) must remain in their location. They must be in the working directory for the program to work as intended.
+
+## Development
+
+### Dependencies
```

Comment:
This is out of scope of this PR, but we could investigate using something like [`requirements.txt`](https://note.nkmk.me/en/python-pip-install-requirements/) to make it easier to install all dependencies. 

I haven't written python in a while, so there may be a more modern option as well 🤷